package.json

```diff
@@ -64,7 +65,7 @@
     "tailwind": "run-p tailwind:watch sync",
     "tailwind:watch": "npm run build:tailwind -- -w",
     "test": "npm run lint",
-    "watch": "wait-on http://backend:8000/cms && run-s build:clean && run-p build:js:dev build:common watch:**",
+    "watch": "run-s build:clean && run-p build:js:dev build:common watch:**",
```

Comment:
The frontend build was rather slow, so I removed the wait-on to speed things up a bit.